modules/displaying-potential-issues-of-your-cluster.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-of-your-cluster.adoc
+
+[id="displaying-potential-issues-of-your-cluster_{context}"]
+= Displaying potential issues of your cluster
+
+This section describes how to display the Insights report the {cloud-redhat-com} has created.
```

Comment:
This translates to statement that OCM generates the Insights report for you which is factually wrong and might be confusing in the future when OCM is not the only UI we use. My choice of words would be something in lines of "Insights generates the report, OCM displays it".